### ISSUE TITLE
Chem: 'NullError: method not found: 'K' on null' error when selecting a molecular column in dialog

### ISSUE THREAD

**Issue by ahadzhanian** [datagrok-ai/public#1483]
Platform: Release v. 1.12.0 

Data: [spgi-100](https://release.datagrok.ai/f/System.AppData/Chem/tests/spgi-100.csv)
To reproduce:

1. On the menu ribbon, open **Chem** | **Activity cliffs**. A dialog opens.
2. In **Molecules**, select the **Structure** column. An error occurs

````
html_dart2js.dart:3558 NullError: method not found: 'K' on null
	https://release.datagrok.ai/packages/d4/src/common/column_combo_box.dart 388:69                    bhl.
	https://release.datagrok.ai/packages/$sdk/lib/async/stream_pipe.dart 201:19                        aC.B8
	https://release.datagrok.ai/packages/$sdk/lib/async/stream_pipe.dart 164:5                         Nt.aES
	https://release.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/js_helper.dart 2811:12      Ud.eval
	https://release.datagrok.ai/packages/$sdk/lib/html/dart2js/html_dart2js.dart 42920:48              d7W.
	https://release.datagrok.ai/packages/$sdk/lib/async/zone.dart 1132:12                              aWR
	https://release.datagrok.ai/packages/$sdk/lib/async/zone.dart 1012:12                              d5E.zX
	https://release.datagrok.ai/packages/$sdk/lib/async/zone.dart 909:14                               d5E.GX
	https://release.datagrok.ai/packages/$sdk/lib/async/zone.dart 936:23                               d5H.
	https://release.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/js_helper.dart 2456:48      fz1.
	https://release.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/isolate_helper.dart 474:16  Ny.L5
	https://release.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/isolate_helper.dart 59:16   Object.FI
	https://release.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/js_helper.dart 2456:14      fz.
	https://release.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/js_helper.dart 2476:14      HTMLDocument.<fn>
````

![Image](https://user-images.githubusercontent.com/111368431/212674899-376ef013-25c0-4804-a263-2b1d47004b14.gif)

**Comment by ahadzhanian** [comment]
Sometimes can be reproduced on **Chem | Chemical Space** under the same conditions

**Comment by ahadzhanian** [comment]
NullError: method not found: 'K' on null occurs where a molecular column needs to be selected in a dialog or property

**Comment by dionissqq** [comment]
closed in v1.12.0
